### DISCUSSION TITLE
freeRouting with KiCad Issues

### DISCUSSION THREAD

**Discussion by CalvinJC19** [freerouting/freerouting#60] [Q&A]
I am currently doing a project on KiCad where I am currently up to creating the board layout, components and tracks. I have been trying to get the freeRouting software to work however when I open the .dsn file in freeRouting it bugs out and half open the freeRouting software. I have been trying to find answers for this on the internet but have had no luck.

I am running Mac 11.4 and I think freeRouting 1.4 with KiCad 5.1.10. 

Below will be screen shots of the board layout and the freeRouting file half opening:
<img width="737" alt="Screen Shot 2021-08-18 at 9 37 02 pm" src="https://user-images.githubusercontent.com/89137530/129891721-2c95aca6-f54d-44ea-977e-ec607de6e74c.png">
<img width="1148" alt="Screen Shot 2021-08-18 at 9 38 14 pm" src="https://user-images.githubusercontent.com/89137530/129891764-553d532a-7fab-4422-b222-bd219873badf.png">

any help to try and open this .dsn file at this point would be helpful

**Comment by AlexIII** [comment]
Hi! Attach your `.dsn` file, I can try it on my machine.

**Reply by CalvinJC19** [reply]
[PE Individual Project.dsn copy.zip](https://github.com/freerouting/freerouting/files/7010660/PE.Individual.Project.dsn.copy.zip)

**Comment by lynchaj** [comment]
Yes, post .dsn file and I'll try with 1.2.43 Extra

**Reply by CalvinJC19** [reply]
[PE Individual Project.dsn copy.zip](https://github.com/freerouting/freerouting/files/7010659/PE.Individual.Project.dsn.copy.zip)

**Comment by CalvinJC19** [comment]
[PE Individual Project.dsn copy.zip](https://github.com/freerouting/freerouting/files/7010655/PE.Individual.Project.dsn.copy.zip)
 attached is the .dsn files of the PCB

**Comment by lynchaj** [comment]
[PE Individual Project.zip](https://github.com/freerouting/freerouting/files/7010876/PE.Individual.Project.zip)

Hi
Works fine on Win 10, 32 bit FreeRouting 1.2.43 Extra.  Solves quickly with zero vias.  Pretty nice!

Doesn't seem to have any power traces though.  That's odd and very few connections too.

Maybe I am getting only half the file also.  To me that would suggest a problem in parsing the .dsn file

![image](https://user-images.githubusercontent.com/86925539/129988935-d079e761-fe7e-4b63-b5db-8712f453076c.png)

**Comment by lynchaj** [comment]
Hi
It's not clear to me what's missing.  Have you updated the net file in eeschema and reloaded the net in PCBnew?  All of the components loaded seem to be connected in some fashion.  Not sure what it is supposed to do so hard to say if this is sufficient.

Make sure you have an up-to-date net in your PCBnew before you export the .dsn file.  Hope this helps.  Good luck!

**Comment by lynchaj** [comment]
Hi,
Any updates on this issue?

Thanks, Andrew Lynch

**Comment by Betureee** [comment]
Anche io sto usando KiCAD con il MacBook, a me aveva dato quel problema della pagina bianca perché non avevo installato l'ultima versione di java

English translation:
`I'm also using KiCAD with the MacBook, it gave me that blank page problem because I didn't install the latest version of java`
